src/Admin.php

```diff
@@ -22,6 +22,10 @@ class Admin {
   public static function init() {
     // Adds custom fields for single products.
     add_action('woocommerce_product_options_general_product_data', __NAMESPACE__ . '\WooCommerce::woocommerce_product_options_general_product_data');
+    // Adds incoming stock WooCommerce to simple products inventory.
+    add_action('woocommerce_product_options_stock_fields', __NAMESPACE__ . '\WooCommerce::showIncomingstock', 9);
+    // Adds incoming stock WooCommerce to variable products inventory.
+    add_action('woocommerce_product_after_variable_attributes', __NAMESPACE__ . '\WooCommerce::showIncomingstock', 9, 3);
```

Comment:
I don't understand that comment. 🤔 

Also since two hooks are doing the same I'd rather write a single comment
```suggestion
    // Adds product option to show incoming stock.
    add_action('woocommerce_product_options_stock_fields', __NAMESPACE__ . '\WooCommerce::showIncomingstock', 9);
    add_action('woocommerce_product_after_variable_attributes', __NAMESPACE__ . '\WooCommerce::showIncomingstock', 9, 3);
    
```
Is the 2 argument needed in the latter hook? The method seems not to support any parameters.

Reply:
@fabianmarz I had to move the second one, as requested by Stephan to show in different order than what was possible like so, please review again.